contracts/test-alt/_marketplaceHelpers.js

```diff
@@ -47,7 +47,7 @@ export default function({
     }
   }
 
-  async function makeOffer({ withdraw, listingID = 0 }) {
+  async function makeOffer({ withdraw, listingID = 0, affiliate = Buyer }) {
```

Comment:
I realize that was its value before, but I still don't understand the reasoning behind the value

Reply:
Initially it was so that the buyer can earn their own commission as a kind of 'reward points'. We could also make another account called 'Affiliate' and use that instead, to make the tests clearer